sessions/class-01/exercise/start.js

```diff
@@ -34,7 +48,10 @@ export function formatCurrency(num, decimal) {}
  * @param {number} percentage Discount percentage (like 25 for 25% discount).
  * @returns {number} formatted (2 decimal precision) value of discounted price.
  */
-export function getDiscountedValue(price, percentage) {}
+export function getDiscountedValue(price, percentage) {
+    let discountedPrice = safelyConvertToNumber(price) * (safelyConvertToNumber(percentage) / 100);
+    return formatCurrency(price - discountedPrice,2);
```

Comment:
Here also we do not really need to assign `safelyConvertToNumber` to a variable. We could have just returned in a single statement.